book/005-asynchronous-tooling.md

```diff
@@ -274,23 +274,23 @@ fetch('/some-endpoint/users')
   );
 ```
 
-There's nothing wrong with either approach, just make sure you and your team agrees on which method is preferred.
+There's nothing wrong with either approach, just make sure you and your team agree on which method is preferred.
 
-**IMPORTANT** As of Node v10, Unhandled rejected promises will halt and/or crash the Node process. If you aren't handling rejected promises now, you will need to it soon, otherwise your whole program will crash.
+**IMPORTANT** As of Node v10, Unhandled rejected promises will halt and/or crash the Node process. If you aren't handling rejected promises now, you will need to soon, otherwise your whole program will crash.
 
 ### Keep Promises Simple
 
 Promises open up so much in async functionality, making callback hell a thing of the past and multiple REST API calls easy to read and move through.
 
-Because things get easier, the want to create more complex functions grows larger and larger, it's even more imperative that the **Single responsibilty principle** is upheld. For those needing a refresher:
+Because things get easier and the want to create more complex functions grows larger and larger, it's even more imperative that the **Single responsibilty principle** is upheld. For those needing a refresher:
```

Comment:
Gahhh.. Tense...

```suggestion
Because things are getting easier, and the want to create more complex functions grows larger and larger, it's even more imperative that the **Single responsibilty principle** is upheld. For those needing a refresher:
```